docs/reference/pyproject_toml.md

```diff
@@ -0,0 +1,241 @@
+# Pyproject.toml
+
+`pyproject.toml` is a
+[standardized](https://packaging.python.org/en/latest/specifications/pyproject-toml/) file for
+specifying the metadata and build system of a Python project. See [Projects](../guides/projects.md)
+for an introduction.
+
+Most parts of uv only consider the name, version, (optional) dependencies and build system of a
+project, and read only those fields from `pyproject.toml`. The `project.name` is always required,
+while `project.version`. If `project.dependencies` is not specified, it means that the project has
+no dependencies. If you need to use dynamic dependencies (discouraged), you must add `dependencies`
+to `project.dynamic`. The same applies to `optional-dependencies`.
+
+For the build backend (`build-system = "uv"`), all fields are relevant and get translated to
+[Core Metadata](https://packaging.python.org/en/latest/specifications/core-metadata) in the final
+field. uv supports the
+[living standard](https://packaging.python.org/en/latest/specifications/core-metadata) in addition
+to the provisional [PEP 639](https://peps.python.org/pep-0639/) for better upload metadata. When
+using the uv build backend, the `project.name`, `project.version`, `build-system.requires` and
+`build-system.build-backend` keys are required, `project.dynamic` is not supported, and all other
+fields are optional.
+
+## The `[build-system]` table
+
+The may be breaking changes to the uv build backend configuration in future uv versions, so you
```

Comment:
There are typos -- `The` -> `There` and `so you constrain` -> `so you should constrain`.